backend/src/domain/contracts/repos/rating.ts

```diff
@@ -0,0 +1,38 @@
+export interface CreateRating {
+    create: (input: CreateRating.Input) => Promise<CreateRating.Output>
+  }
+  
+export namespace CreateRating {
+    export type Input = {
+        requesterId: number,
+        value: number,
+        committee: string,
+        comment: string,
+        ticketId: number
+    }
+    export type Output = {
+        id: number,
+        userId: number,
+        value: number,
+        committee: string,
+        comment: string,
+        ticketId: number
+    }
+}
+export interface LoadRatingByReqId {
+    loadByReqId: (input: LoadRatingByReqId.Input) => Promise<LoadRatingByReqId.Output>
+}
+    
+export namespace LoadRatingByReqId {
+    export type Input = {
+        requesterId: number
+    }
+    export type Output = {
+        id: number,
+        userId: number,
+        value: number,
+        committee: string,
+        comment: string,
+        ticketId: number
+    }
+}
```

Comment:
Aqui foi uma questão de lint, todo arquivo tem que ter uma linha em branco no fim se não o Git reclama. E utilizei identação com tabs e 2 espaços

```suggestion
export interface CreateRating {
  create: (input: CreateRating.Input) => Promise<CreateRating.Output>
}

export namespace CreateRating {
  export type Input = {
    requesterId: number
    value: number
    committee: string
    comment: string
    ticketId: number
  }
  export type Output = {
    id: number
    userId: number
    value: number
    committee: string
    comment: string
    ticketId: number
  }
}
export interface LoadRatingByReqId {
  loadByReqId: (
    input: LoadRatingByReqId.Input
  ) => Promise<LoadRatingByReqId.Output>
}

export namespace LoadRatingByReqId {
  export type Input = {
    requesterId: number
  }
  export type Output = {
    id: number
    userId: number
    value: number
    committee: string
    comment: string
    ticketId: number
  }
}

```